doc/references.rst

```diff
@@ -5,18 +5,18 @@ References
     When adding references here, please use the demonstrated format:
     [FirstAuthor_pubyear]
 
-.. [Shewchuk_2002] J. R. Shewchuk (2002), \
-    What Is a Good Linear Finite Element? - \
-    Interpolation, Conditioning, Anisotropy, and Quality Measures, \
-    In Proc. of the 11th International Meshing Roundtable \
-    `(link) <http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.19.2164>`__
+.. [Shewchuk_2002] J. R. Shewchuk (2002),
+    *What Is a Good Linear Element? - Interpolation, Conditioning, Quality Measures*,
+    In Proceedings of the 11th International Meshing Roundtable.
+    `(link) <https://hal.science/hal-04614934/>`__
```

Comment:
Citeseerx seems to be mostly dead these days, so this uses hal instead (hopefully it's the same paper.. the title differs slightly)